templates/solutions/ai/edge.html

```diff
@@ -0,0 +1,272 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/17eHoiLrhFLrfV8UXD_4bxn-yY1_YToxf4cs8-5kYkxo/edit
+{% endblock meta_copydoc %}
+
+{% block title %}Enterprise AI edge computing{% endblock %}
+
+{% block meta_description %}
+  Get everything you need to deploy AI at the edge in one solution. Secure and compliant open source optimised for embedded AI use cases.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-suru--50-50 p-section--shallow">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="p-section--shallow">
+          Develop AI models and run
+          <br />
+          them at the edge &mdash; all on the
+          <br />
+          same open source stack
+        </h1>
+      </div>
+      <div class="col">
+        <p>
+          Edge computing and AI go hand-in-hand to drive some of today's most valuable enterprise use cases. Canonical delivers a single stack with everything you need to easily and securely deploy and manage AI at the edge. Unlock the power of real-time, AI-driven insight in the field.
+        </p>
+        <hr />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="ai-modal">Contact Canonical</a>
```

Comment:
```suggestion
           aria-controls="ai-modal">Contact Canonical <span class="u-off-screen">about AI solutions for edge computing</span></a>
```
This is something I have started to do on links (after you told me about it haha). It isn't necessary but I think adds a nice QOL for people using screen readers, even when there is context

Reply:
Agreed, thanks for flagging!